sdks/typescript/src/apache_beam/runners/portable_runner/runner.ts

```diff
@@ -79,6 +79,17 @@ class PortableRunnerPipelineResult implements PipelineResult {
     return state;
   }
 
+  async cancel() {
```

Comment:
does it make sense to pass a completion callback to the cancel function itself?

Reply:
feel free to add as TODO : )

Reply:
I'm not sure there's much benefit in doing so (over just using await). If so, we can always add it later (but it wouldn't be possible to remove).